src/App.css

```diff
@@ -101,28 +101,54 @@ h3 {
 
 .opentype-label {
   font-family: monospace;
+  border-radius: 0.2rem;
   display: inline-flex;
   align-items: center;
   height: 1.5rem;
   padding: 0 0.75rem;
   margin-right: var(--small-margin);
-  border-radius: 99999px;
   background: var(--offwhite);
 }
 
-.beta-message {
-  position: fixed;
-  z-index: 200;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  padding: 0.5em 2em;
-  text-align: center;
-  background: var(--yellow);
+.button {
+  padding: 0.25rem 1rem;
+  border-radius: 99999px;
+  margin: 0;
+  border: 0;
+  background: none;
+  font-size: 0.85rem;
+  background: var(--unlighter-grey);
+  cursor: pointer;
+  min-width: 6em;
+  user-select: none;
+  color: black;
 }
 
-.beta-message a {
-  color: inherit;
+.button:hover {
+  box-shadow: inset 0 0 999px rgba(0, 0, 0, 0.15);
+}
+
+.button[disabled] {
+  background: var(--unlighterer-grey);
+}
+
+.button.active {
+  background: var(--dark-grey);
+  color: white;
+}
+
+.button.active:hover {
+  box-shadow: inset 0 0 999px rgba(0, 0, 0, 0.4);
+}
+
+.button.on {
+  background: var(--green);
+  color: white;
+}
+
+.button.off {
```

Comment:
Looking at its use I'm not understanding the thought process behind calling the variants „on” and „off”.
Could you enlighten me?

Reply:
It's to represent the state of a button where it's used to turn a feature on or off.